src/test/rgw/rgw_multi/tests.py

```diff
@@ -661,6 +662,14 @@ def test_bucket_acl():
         bucket.set_acl('public-read')
         assert(len(bucket.get_acl().acl.grants) == 2) # new grant on AllUsers
 
+def test_bucket_cors():
+    buckets, zone_bucket = create_bucket_per_zone_in_realm()
+    for zone, bucket in zone_bucket.items():
```

Comment:
after rebasing this on master, i'm seeing this failure:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/usr/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/util.py", line 620, in newfunc
    return func(*arg, **kw)
  File "/home/cbodley/ceph/src/test/rgw/rgw_multi/tests.py", line 660, in test_bucket_cors
    for zone, bucket in zone_bucket.items():
AttributeError: 'list' object has no attribute 'items'
```
commit f5037624fd4a96e350747be3355d9ab24114189a changed `zone_bucket` from dict to list, so this line needs to be `for zone, bucket in zone_bucket:`